### PR TITLE
Use the custom robot from SimulatorCore

### PR DESCRIPTION
**Pull request by Casper-Oakley**
Just put in the import line for the robot. Included the 120KB robot sent around via email, looks good to me as is.

I guess there's still an open question about grabbers. https://github.com/FRUK-Simulator/SimulatorCore/pull/127/files has the grabber as a bucket facing in front of the robot, leaving the arm at the top of the FRUK robot looking a little out of place

Source: https://github.com/actions/setup-node/issues/212

### CODE REVIEW

File: src/RobotSimulator/StdWorldBuilder.ts [hunk 1]
```diff
@@ -75,6 +75,20 @@ export class StdWorldBuilder {
 
     const robotSpec = robotBuilder.generateSpec();
     robotSpec.initialPosition = this.startPosition;
+    robotSpec.customMesh = {
+      filePath: `${process.env.PUBLIC_URL}/assets/models/robot_full_compressed.glb`,
```
Comment (JamesWP) [inline src/RobotSimulator/StdWorldBuilder.ts:79]: `process.env.PUBLIC_URL` how does that work? there are no environment variables in the web browser?

File: src/RobotSimulator/StdWorldBuilder.ts [hunk 2]
```diff
@@ -75,6 +75,20 @@ export class StdWorldBuilder {
...
+      rotation: {
+        x: 0,
+        y: 0,
+        z: 0,
+      },
+      scale: {
+        x: 0.2,
```
Comment (JamesWP) [inline src/RobotSimulator/StdWorldBuilder.ts:86]: did you check the sizes of the model matches this? the debug mode should help you decide this

**Review by JamesWP — COMMENTED** [review]
Just a few questions before we land this

File: src/RobotSimulator/StdWorldBuilder.ts (hunk 1 above)
Comment (Casper-Oakley) [reply src/RobotSimulator/StdWorldBuilder.ts:79]: https://create-react-app.dev/docs/using-the-public-folder/ - react essentially replaces it with the public folder location

File: src/RobotSimulator/StdWorldBuilder.ts (hunk 2 above)
Comment (Casper-Oakley) [reply src/RobotSimulator/StdWorldBuilder.ts:86]: Nope, will do now

**Comment by JamesWP** [comment]
Closing this for now. Might be useful later

**Pull request closed without merging**